docs/release-notes.md

```diff
@@ -22,6 +22,8 @@ nav_order: 9
 - Drop `LuksOption` and `RaidOption` types _(Go API for fcos 1.5.0-experimental,
   flatcar 1.1.0-experimental, openshift 4.13.0-experimental)_
 - Require Go 1.18+
+- Correctly document `hash`, and clarify how its effected by the `compression`
```

Comment:
This should be under `Docs changes`.

How about:

```
- Document that `hash` fields describe decompressed contents
- Clarify spec docs for `files`/`luks` `hash` fields
```

Reply:
Sigh, I should have made sure there was not a doc's section. Sorry about that. And yes those sound better.
Ill make a follow up pr for ignition as well.